packages/amplication-client/src/Project/ArchitectureConsole/hooks/useModelOrganizer.tsx

```diff
@@ -134,7 +134,7 @@ const useModelOrganizer = ({
 
   const loadProjectResources = useCallback(
     (forceRefresh?: boolean, onLoadResourcesCompleted?: () => void) => {
-      if (!forceRefresh) {
+      if (!forceRefresh && !headlessMode) {
```

Comment:
```suggestion
      // do not load the saved data in haedless mode - this will override any saved data when used as headless 
      if (!forceRefresh && !headlessMode) {
```